omero/users/cli/installation.rst

```diff
@@ -24,6 +24,9 @@ Alternatively install ``omero-py`` using venv::
     . myenv/bin/activate
     pip install omero-py
 
+.. Note:: On Ubuntu 20.09, you may need to install ``libssl-dev`` before running the above command.
```

Comment:
Maybe 
```suggestion
.. Note:: On Ubuntu 20.09, you may need to install ``libssl-dev`` before installing CLI.
```

Alternatively, push this note above the two blocks of commands (one for conda, one for venv) and say

``On Ubuntu 20.09, you may need to install ``libssl-dev`` before running the commands below.``

Otherwise, it is not very clear which command to run first, and which next.